### PR TITLE
add meta tags

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: theme/index.hbs [hunk 1]
```diff
@@ -12,6 +12,11 @@
     <meta name="description" content="{{ description }}">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="theme-color" content="#ffffff" />
+    <meta property="og:title" content="{{ title }}">
+    <meta property="og:image" content="{{ path_to_root }}{{ favicon }}">
+    <meta property="og:description" content="{{ description }}">
+    <meta property="og:url" content="https://alan-lang.org">
```
Comment (dfellis) [inline theme/index.hbs:18]: This should be https://docs.alan-lang.org

File: theme/index.hbs [hunk 2]
```diff
@@ -12,6 +12,11 @@
     <meta name="description" content="{{ description }}">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="theme-color" content="#ffffff" />
+    <meta property="og:title" content="{{ title }}">
+    <meta property="og:image" content="{{ path_to_root }}{{ favicon }}">
```
Comment (dfellis) [inline theme/index.hbs:16]: Same question on the "proper" format for this image.

File: theme/css/variables.css [hunk 3]
```diff
@@ -90,7 +90,7 @@
 }
 
 .solarized-dark {
-    --bg: var(--base03);
+    --bg: var(--base02);
```
Comment (depombo) [inline theme/css/variables.css:93]: after:
Screen Shot 2020-08-11 at 1 05 06 PM

before:
Screen Shot 2020-08-11 at 1 05 13 PM

File: theme/css/variables.css [hunk 4]
```diff
@@ -63,8 +63,8 @@
 }
 
 .solarized-light {
-    --bg: var(--base3);
```
Comment (depombo) [inline theme/css/variables.css:66]: after:
Screen Shot 2020-08-11 at 1 07 40 PM

before:
Screen Shot 2020-08-11 at 1 23 48 PM

**Pull request closed without merging**